paasta_tools/list_kubernetes_service_instances.py

```diff
@@ -88,9 +102,25 @@ def main():
         else:
             app_name = compose_job_id(name, instance)
         service_instances.append(app_name)
-    print("\n".join(service_instances))
+    if args.shuffle:
+        random.shuffle(service_instances)
+    if args.group_lines:
+        group_lines(service_instances, args.group_lines)
+    else:
+        print("\n".join(service_instances))
```

Comment:
suggestion:

```suggestion
    if args.shuffle:
        random.shuffle(service_instances)
        
    if args.group_lines:
        group_lines(service_instances, args.group_lines)
    else:
        print("\n".join(service_instances))
```

would make it clearer that these aren't mutually exclusive - without the whitespace, it's not immediately clear that one can hit both cases (thanks to the magic of human pattern matching :p)